### PR TITLE
cloud - Centralised Observability

### PR DESCRIPTION
**Pull request by danieldides**

### CODE REVIEW

**Review by michaellzc — APPROVED** [review]

File: content/departments/cloud/index.md [hunk 1]
```diff
@@ -290,3 +290,21 @@ For #ce teammates, please reach out to #cloud and include link to this FAQ
 
 Cloud supports all code-hosts types (self-managed and Cloud-managed), but it currently requires the code-host to have a public IP.
 More context [here](https://docs.google.com/document/d/14S3jn0bV03WdeT1H36omvtGJFoIFJjM-3ZA1qIyIl7o/edit)
+
+### FAQ: How do I figure out the GCP Project ID for a customer?
+
+The best way to determine the project ID for a given customer is to look up the customer in the `deploy-sourcegraph-managed` repo using the following query on S2:
+
+```
+repo:^github\.com/sourcegraph/deploy-sourcegraph-managed$ file:config.yaml <CUSTOMER NAME OR DOMAIN HERE>
```
Comment (bobheadxi) [inline content/departments/cloud/index.md]:
```suggestion
repo:^github\.com/sourcegraph/deploy-sourcegraph-managed$ file:config\.yaml lang:yaml customer: :[_\n]
```

https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph-managed$+file:config%5C.yaml+lang:yaml+customer:+:%5B_%5Cn%5D&patternType=structural&sm=1&groupBy=path

image

File: content/departments/cloud/technical-docs/observability/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Cloud Observability
+
+Epic link: https://github.com/sourcegraph/customer/issues/1151
+
+## Metrics
+
+Metrics are gathered from all resources using the included Prometheus instance. This instance scrapes and stores the metrics locally as well as forwards them to the Managed Prometheus service provided by GCP. These metrics are viewable through our centralised Grafana instance hosted at: https://monitoring.sgdev.org.
+
+> Note: access to these resources must be granted. To request access, follow the [Requesting access to Grafana](./operations.md#requesting-access-to-grafana).
```
Comment (bobheadxi) [inline content/departments/cloud/technical-docs/observability/index.md]: To use the built-in admonition thing:

```suggestion
> NOTE: access to these resources must be granted. To request access, follow the [Requesting access to Grafana](./operations.md#requesting-access-to-grafana).
```

File: content/departments/cloud/technical-docs/observability/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Cloud Observability
+
+Epic link: https://github.com/sourcegraph/customer/issues/1151
+
+## Metrics
+
+Metrics are gathered from all resources using the included Prometheus instance. This instance scrapes and stores the metrics locally as well as forwards them to the Managed Prometheus service provided by GCP. These metrics are viewable through our centralised Grafana instance hosted at: https://monitoring.sgdev.org.
```
Comment (bobheadxi) [inline content/departments/cloud/technical-docs/observability/index.md]:
```suggestion
Metrics are gathered from all resources using the included Prometheus instance. This instance scrapes and stores the metrics locally as well as forwards them to the Managed Prometheus service provided by GCP.

**Only metrics queried in our [monitoring generator](https://docs.sourcegraph.com/dev/background-information/observability/monitoring-generator) are forwarded - this allowlist is automatically generated.** If you'd like a new metric to be queryable in a centralized manner, you _must_ [create a dashboard panel](https://docs.sourcegraph.com/dev/how-to/add_monitoring#alerts-dashboards-and-documentation) for it.

These metrics are viewable through our centralised Grafana instance hosted at: https://monitoring.sgdev.org.
```

File: content/departments/cloud/index.md [hunk 4]
```diff
@@ -290,3 +290,21 @@ For #ce teammates, please reach out to #cloud and include link to this FAQ
 
 Cloud supports all code-hosts types (self-managed and Cloud-managed), but it currently requires the code-host to have a public IP.
 More context [here](https://docs.google.com/document/d/14S3jn0bV03WdeT1H36omvtGJFoIFJjM-3ZA1qIyIl7o/edit)
+
+### FAQ: How do I figure out the GCP Project ID for a customer?
+
+The best way to determine the project ID for a given customer is to look up the customer in the `deploy-sourcegraph-managed` repo using the following query on S2:
+
+```
+repo:^github\.com/sourcegraph/deploy-sourcegraph-managed$ file:config\.yaml lang:yaml customer: :[_\n]
```
Comment (filiphaftek) [inline content/departments/cloud/index.md:299]: @danieldides sorry that so late I review it, but this won't work for PLG automated trials - it should be grep by emailDomain, b/c customer names are random 🙂  but the second option with gcloud and `domain` field works for all 👍

Comment (filiphaftek) [reply content/departments/cloud/index.md:299]: Great docs btw 👏

**Pull request merged**